modules/monitoring-about-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-creating-alerting-rules-for-user-defined-projects_{context}"]
+= About creating alerting rules for user-defined projects
+
+If you create alerting rules for a user-defined project, consider the following key behaviors and important limitations when you define the new rules:
+
+* You can only create new alerting rules for user-defined projects.
```

Comment:
I think we can omit this paragraph as it can be confusing, because we can define alerting rules to platform monitoring using `AlertingRule` (as explained in `Managing alerting rules for core platform monitoring`)

Reply:
Deleted the paragraph.